json_templates/azure/c0-azure-base/ert-template.json

```diff
@@ -1,14 +1,14 @@
 {
-  "networks_and_azs":{
+  "networks_and_azs": {
     "networks_and_azs": {
       "singleton_availability_zone": {
-      "name": "null"
-    },
-    "other_availability_zones": [
-      {
         "name": "null"
-      }
-    ],
+      },
+      "other_availability_zones": [
+        {
+          "name": "null"
```

Comment:
@krishicks not sure why it was null. We can create a new bug or story in the tracker to get to this. The scope of this story was just to support the azure blob store for ERT tile.